cli_reference/basic_cli_operations.adoc

```diff
@@ -226,6 +226,43 @@ link:../dev_guide/deployments.html[deployment configurations] using standard
 `--from-build=_<build_name>_ --follow`
 |Manually start the build process by specifying either a configuration file or the name of a previous build and retrieves its build logs.
 
+|`oc start-build \`
+`--from-build=_<build_name>_ --wait`
+|Wait for a build to complete and exit with a non-zero return code if the build fails.
+
+|`oc start-build --env *(-e)* FOO=bar`
+|Set or override environment variables for the current build. Does not change the BuildConfig.
+
+|`oc start-build --build-loglevel`
+|Set or override the default build log level output [0-5] during the build.
+
+|`oc start-build --commit`
+|Specify the source code commit identifier the build should use; requires a build based on a Git repository.
+
+|`oc start-build --from-build`
+|Specify the name of a build which should be re-run.
+
+|`oc start-build --from-dir`
+|A directory to archive and use as the binary input for a build.
```

Comment:
See `--from-build` comment with replaceable `<dir>`.  Suggest: "Archive `<dir>` and build with it as the binary input."